app/controllers/api/v1/application_controller.rb

```diff
@@ -25,9 +25,13 @@ def authentication_error
       render json: { error: I18n.t('errors.401') }, status: 401
     end
 
-    def record_not_found_error
+    def existence_error
       render json: { error: I18n.t('errors.404') }, status: 404
     end
 
+    def conflict_error
+      render json: { error: I18n.t('errors.409') }, status: 409
+    end
```

Comment:
some would argue you should only have methods in `ApplicationController` that are shared among more than one other controller. Looks like this isn't used outside of `GamesController` at the moment, maybe others in the future? I leave the decision to you if you want to move it.